src/main/java/tutorspet/logic/util/LessonUtil.java

```diff
@@ -26,6 +28,46 @@
  */
 public class LessonUtil {
 
+    /**
+     * Returns a {@code Lesson} where the {@code Attendance}s of the {@code studentToRemove} has been removed.
+     */
```

Comment:
Not sure if this is correct but I referred to ur previous javadoc for inspiration...
```suggestion
    /**
     * Returns a {@code Lesson} where the {@code Attendance}s of the {@code studentToRemove} have been removed.
     */
```
``